src/components/common/Error/Error.tsx

```diff
@@ -0,0 +1,7 @@
+export const ErrorMessage = (message: { message: string }) => {
+  return <div>{message.message}</div>;
+};
+
+export const EmptyMessage = () => {
+  return <div>데이터가 없는데용?</div>;
+};
```

Comment:
```suggestion
export const ErrorMessage = (children: PropsWithChildren) => {
  return <div>{children ?? '데이터가 없는데용?'}</div>;
};
```

이렇게 표현할 수 있지 않을까 싶네요!